PMMain/Midas/MidasGUI/MyoTranslationFilter.cpp

```diff
@@ -35,24 +37,32 @@ void MyoTranslationFilter::process()
         baseYaw = yaw;
     }
 
+    outputToSharedCommandData[DELTA_VOL] = 0;
     if (controlStateHandle->getMode() != MOUSE_MODE)
     {
         if (previousMode == MOUSE_MODE)
         {
-            filterDataMap outputToSharedCommandData;
             point mouseUnitVelocity = point(0, 0);
             outputToSharedCommandData[VELOCITY_INPUT] = mouseUnitVelocity;
-            Filter::setOutput(outputToSharedCommandData);
+        }    
+
+        // Jorden TODO -- put other "if (controlStateHandle->getMode() == XXX)" statements here, and generate output
+        // to populate the sharedcommanddata with. Such as rotational data for volume
+        // temp
+        if (controlStateHandle->getMode() == GESTURE_HOLD_THREE)
```

Comment:
I know right now you have GESTURE_HOLD_N where N is a number, for each type of gesture that can be held. If we are going to go down this route (have a state for each held gesture-- as I mentioned in the last CR, less maintainable but easy to implement), should we have a slightly more specific name so it is easier to remember what each one does? So, instead of GESTURE_HOLD_THREE, have GESTURE_HOLD_FIST, etc...